test/performance/performance.go

```diff
@@ -152,3 +158,69 @@ func sanitizedURL(endpoint string) string {
 	}
 	return endpoint
 }
+
+// DeploymentStatus is a struct that wraps the status of a deployment.
+type DeploymentStatus struct {
+	DesiredReplicas int32
```

Comment:
Perhaps we should include the timestamp for the tick?

Reply:
Added.